### ISSUE TITLE
Set up postgres continuous archiving

### ISSUE THREAD

**Issue by aapeliv** [Couchers-org/couchers#1358]
https://www.postgresql.org/docs/current/continuous-archiving.html

**Comment by kalvdans** [comment]
What is the threat we want to protect us against? Accidental removal of large chunks of the database? Natural disasters? Amazon going bankrupt?

**Comment by aapeliv** [comment]
Different types of user error mostly: e.g. accidentally deleting things (on any level). Also things like the server failing (EC2 instance themselves can fail, and are no more reliable than other servers).

It's more efficient to do continuous archiving and take snapshot backups more rarely than it is to take snapshot every upgrade as we do now.

**Comment by kalvdans** [comment]
To protect from server failure we also need to move the backups to another host.

**Comment by aapeliv** [comment]
Yeah, they're currently being sent to an S3 bucket, which is much more reliable than a server (it's distributed over multiple disks and does some secret AWS magic). I'm envisioning the continuous archiving being the same.
